js/ajax_settings.js

```diff
@@ -11,9 +11,24 @@ onOffice.ajaxSaver = function(outerDiv) {
 (function() {
 	this.register = function() {
 		var proto = this;
-		this._outerDiv.find('#send_ajax').on('click', function() {
-			window.scrollTo(0, 0);
-			proto.save();
+		var form = $("form");
+		var typesNeedValid = ['email'];
+
+		form.submit(function (e) {
+			e.preventDefault();
```

Comment:
I'm a bit confused by this. Why do we need to prevent the form from sending and validate and save it ourselves here? Shouldn't that be something we do in #160?

Reply:
I think it would be better to fix that in #160